spec/steps/discover.fmf

```diff
@@ -45,7 +45,9 @@ description: |
         directly in the plan as a list of dictionaries containing
         test ``name`` and actual ``test`` script. Optionally it is
         possible to define any other :ref:`/spec/tests` attributes
-        such as ``path`` or ``duration`` here as well.
+        such as ``path`` or ``duration`` here as well. The default
+        :ref:`/spec/tests/duration` for tests defined directly in
+        the discover step is ``1h``.
```

Comment:
Right, so it's not just an `execute: \n   script:` thing. Might not be obvious (at least for me).

Reply:
Not sure about your question: This is part of the [discover step specification](https://tmt.readthedocs.io/en/latest/spec/steps.html#shell) which includes all related information and examples. What exactly are you missing?

Reply:
It was just an additional note (nothing is wrong), because I would have thought these have "L1 default duration" set, because of the way I'm entering them into the format.

But as long as the statement is clear (somewhere [here](https://tmt.readthedocs.io/en/latest/spec/steps.html?highlight=duration#discover)) - then everything is fine (I'm unsure whether `spec/steps/discover.fmf` corresponds to the document I've linked).

Reply:
I see, thanks for clarification.